sycl/include/sycl/detail/pi.h

```diff
@@ -313,6 +315,8 @@ typedef enum {
   PI_DEVICE_INFO_ATOMIC_64 = 0x10110,
   PI_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES = 0x10111,
   PI_DEVICE_INFO_ATOMIC_MEMORY_SCOPE_CAPABILITIES = 0x11000,
+  PI_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES = 0x10114,
+  PI_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES = 0x10115,
```

Comment:
could you rename these and use contiguous values from 0x1FFFF just below
 ```suggestion
  PI_EXT_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES = 0x10114,
  PI_EXT_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES = 0x10115,
```

Reply:
The file already has `PI_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES` and `PI_DEVICE_INFO_ATOMIC_MEMORY_SCOPE_CAPABILITIES`. Should I change those to add `EXT` too?

Reply:
It's up to you if you want to fix unrelated misses.